README.md

```diff
@@ -18,97 +18,76 @@ It's easy as:
 git clone https://github.com/theiconic/docker-nginx-akeneo.git
 cd docker-nginx-akeneo
 ```
-2. Then provision the systems. Now, this could take up to half an hour depending on your connection. **TODO.** improve this. Possibly due to slow composer install
+2. Set your environment parameters
```

Comment:
for some reason this is been rendered as item 1, so you have to items 1 in your README.